packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -174,6 +189,65 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
 
   // Internal helper methods:
 
+  // Returns an ExpressionPriceFeed.
+  async function _createExpressionPriceFeed(expressionConfig) {
+    // Build list of configs that could be used in the expression including default price feed configs and customFeeds
+    // that the user has provided inside the ExpressionPriceFeed config. Note: default configs are overriden by
+    // customFeeds with the same name. Tranform keys by stripping "/" since that would be interpreted as division.
+    const allConfigs = Object.fromEntries(
+      Object.entries({ ...defaultConfigs, ...expressionConfig.customFeeds }).map(([key, value]) => {
+        return [key.replace("/", ""), value];
```

Comment:
hold on. just to double-check I understand. I can't put any feed names that have `/` within the feed expression, right? this is just parsing the list of default feeds to match this rule that expression feeds cant contain any `/`?

Reply:
The way it previously worked (and there was a subtle bug :)) was that you would enter an expression, like:
```
"ETHBTC + STABLESPREADUSDC"
```

That would match the two existing identifiers:
```
ETH/BTC
STABLESPREAD/USDC
```
because we strip out the `/` before searching through the default list.

The weird caveat is that it would also match:
```
ETHBTC
ET/HBTC
E/THBTC
...
```

This has now changed to using escape characters. PTAL :)